hpi-schul-cloud/nuxt-client
# BC-6674-Replace-language-keys-for-tldraw

# Short Description

QA note:
Change tested on env:
https://bc-6674-replace-language-keys-for-tldraw.brb.dbildungscloud.dev/
No issues found. Only 2 messages 4404 and 4401 were tested in all languages. The rest messages are difficult to trigger so I just checked the code to see if it matched the expected text. 
See screen: https://ticketsystem.dbildungscloud.de/browse/BC-6674





## Links to Ticket and related Pull-Requests

- https://ticketsystem.dbildungscloud.de/browse/BC-6674



## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
